initramfs/init.sh

```diff
@@ -101,25 +103,44 @@ find_and_run_update()
 			continue
 		fi
 
-		if [ ! -x "${UPDATE_SCRIPT}" ]; then
+		if [ ! -x "${TOOLBOX_IMAGE}" ]; then
```

Comment:
since we decided to remove the file after the update; you can just check if the image is only readable (-r).

Reply:
Hmm no this is intended, in the case removing failed